README.md

```diff
@@ -22,60 +22,60 @@ persistence.
+- **Grounded in [Domain-Driven Design]** — Dogma adopts core DDD concepts to
   guide how applications are decomposed and how messages flow between
   components.
 
-- **High-level testing** – The [testkit] module encourages verification of
+- **High-level testing** — The [testkit] module encourages verification of
   application behavior by making assertions about domain events rather than
   inspecting state. It integrates seamlessly with Go’s standard [testing]
   package.
 
-- **Native introspection** – Dogma's static analysis tools visualize message
+- **Native introspection** — Dogma's static analysis tools visualize message
   flow and application structure, enabling discovery of domain events across
   large codebases and multi-application projects.
 
-- **Domain and infrastructure separation** – Domain logic is cleanly and
+- **Domain and infrastructure separation** — Domain logic is cleanly and
   strictly separated from infrastructure concerns such as message delivery,
   persistence and telemetry.
 
-- **Type-agnostic** – Messages and application state can be any Go type that can
+- **Type-agnostic** — Messages and application state can be any Go type that can
   be marshaled to a byte slice, with built-in support for JSON and Protocol
   Buffers.
 
-- **Flexible persistence** – Support for a range of storage options such as
+- **Flexible persistence** — Support for a range of storage options such as
   PostgreSQL and Amazon DynamoDB, enabling use across diverse environments.
 
 ## Ecosystem
 
 Dogma is a collection of Go modules that together provide the tools needed to
 build, test, analyze, and run message-driven applications.
 
-- [dogma] (this repository) – Defines the API for building applications.
-- [testkit] – Utilities for testing Dogma applications.
-- [projectionkit] – Utilities for building [projections][concepts/projection] in popular database systems.
+- [dogma] (this repository) — Defines the API for building applications.
+- [testkit] — Utilities for testing Dogma applications.
+- [projectionkit] — Utilities for building [projections] in popular database systems.
 
 ### Engines
 
-An important Dogma concept is that of the [engine][concepts/engine] — a Go module embedded within
+An important Dogma concept is that of the [engine] — a Go module embedded within
 your application binary that orchestrates message delivery, state persistence,
 and the execution of application logic.
 
-- [verity] – The original Dogma engine, designed to handle typical application
-  loads in smaller deployments. While production-ready, Verity does not support
-  horizontal scaling of individual applications, using a fail-over model
-  instead.
+- [verity] — The original Dogma engine, designed for typical application loads
+  in smaller deployments. While production-ready, it does not support horizontal
+  scaling of individual applications; it uses a failover model instead.
```

Comment:
`failover model` is it when message could not reach an intended app and it fails over to the next one? Personally, I am a bit confused what failover model is.